### PR TITLE
EZEE-30310: SearchLimit UDW: UDW search limit function parameter

### PR DESCRIPTION
**Pull request by czechmarcin**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-30310
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | yes
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

This PR goes along with https://github.com/ezsystems/ezplatform-admin-ui/pull/904

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/modules/universal-discovery/services/universal.discovery.service.js [hunk 1]
```diff
@@ -195,8 +195,9 @@ export const findLocationsByParentLocationId = (
  * @param {Object} restInfo REST config hash containing: token and siteaccess properties
  * @param {String} query
  * @param {Function} callback
+ * @param {Integer} limit
  */
-export const findContentBySearchQuery = ({ token, siteaccess }, query, callback) => {
+export const findContentBySearchQuery = ({ token, siteaccess }, query, callback, limit) => {
```
Comment (dew326) [inline src/modules/universal-discovery/services/universal.discovery.service.js]: You should provide the default value for BC. `limit = QUERY_LIMIT`

File: src/modules/universal-discovery/services/universal.discovery.service.js [hunk 2]
```diff
@@ -195,8 +195,9 @@ export const findLocationsByParentLocationId = (
  * @param {Object} restInfo REST config hash containing: token and siteaccess properties
  * @param {String} query
  * @param {Function} callback
+ * @param {Integer} limit
```
Comment (tischsoic) [inline src/modules/universal-discovery/services/universal.discovery.service.js]: There is no `Integer` type in JS, only `Number`. :)

**Review by dew326 — CHANGES_REQUESTED** [review]

File: src/modules/universal-discovery/services/universal.discovery.service.js [hunk 3]
```diff
@@ -206,7 +207,7 @@ export const findContentBySearchQuery = ({ token, siteaccess }, query, callback)
                 FacetBuilders: {},
                 SortClauses: {},
                 Query: { FullTextCriterion: query },
-                limit: QUERY_LIMIT,
+                limit: limit,
```
Comment (tischsoic) [inline src/modules/universal-discovery/services/universal.discovery.service.js]: You can do: `limit: limit,` --> `limit,`

**Review by tischsoic — CHANGES_REQUESTED** [review]
Only minor issues from my side. :)

**Review by tischsoic — APPROVED** [review]

**Review by kmadejski — APPROVED** [review]

File: src/modules/universal-discovery/services/universal.discovery.service.js (hunk 1 above)
Comment (czechmarcin) [reply src/modules/universal-discovery/services/universal.discovery.service.js]: Now it's provided

**Review by dew326 — APPROVED** [review]

**Review by konradoboza — APPROVED** [review]

**Review by micszo — APPROVED** [review]

**Comment by lserwatka** [comment]
Could you merge it up?

**Comment by czechmarcin** [comment]
commit 948330a49e3891608fc7dce7ed60abad82115086 (HEAD -> master, tag: v1.5.0-rc3, origin/master, origin/HEAD)
Merge: ff600ae 7734265
Author: Dawid Parafiński <dawid.parafinski@ez.no>
Date:   Mon Mar 18 16:28:16 2019 +0100

    Merge branch '1.4' of https://github.com/ezsystems/ezplatform-admin-ui-modules

**Already merged.**

**Pull request merged**